pynest/nest/lib/hl_api_helper.py

```diff
@@ -402,6 +400,38 @@ def broadcast(item, length, allowed_types, name="item"):
     return item
 
 
+@check_stack
```

Comment:
I ran into a circular inclusion problem otherwise. The `SuppressedDeprecationWarning` class definition fits best here among the helpers. It needs to manipulate the verbosity level. Having the verbosity functions defined in `hl_api_info.py` requires helper to include info an vice-versa. Moving the verbosity function definitions here means that `hl_api_helper.py` does not need to include any other `hl_api_*.py` files.